framework/resources/Datasources/table_stats/analyze_tables.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+source conf/drillTestConfig.properties
+
+set -x
+set -e
+
+${DRILL_HOME}/bin/sqlline -n ${USERNAME} -p ${PASSWORD} -u "jdbc:drill:schema=dfs.drillTestDir;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}"  --run=${DRILL_TEST_DATA_DIR}/Datasources/table_stats/analyze_tables.sql
+
+set +x
```

Comment:
Please remove this - it's not needed as we run each script independently and reseting is not necessary